File: src/main/kotlin/org/ailingo/server/AilingoServerApplication.java
package org.ailingo.server;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.EnableConfigurationProperties;

@SpringBootApplication
@EnableConfigurationProperties
public class AilingoServerApplication {

    public static void main(String[] args) {
        SpringApplication.run(AilingoServerApplication.class, args);
    }

}

File: src/main/kotlin/org/ailingo/server/AilingoServerApplication.kt
package org.ailingo.server

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties

@SpringBootApplication
@EnableConfigurationProperties
class AilingoServerApplication

fun main(args: Array<String>) {
    SpringApplication.run(AilingoServerApplication::class.java, *args)
}